### ISSUE TITLE
Chat: Allow to change the font size for normal text output

### ISSUE THREAD

**Issue by tnlanh99** [microsoft/vscode#254112]
I see there is a config setting to change font size, `chat.editor.fontSize`, but it only changes the code, not the others. Can you add a config for this?

<img width="1912" alt="image" src="https://github.com/microsoft/vscode/assets/38713308/1e52094f-327f-4135-884c-252cd3e1f01e">

**Comment by liam-k** [comment]
> The UI has quite a lot of text in it in some places. I think the solution for this is to zoom, I don't want to have separate font settings for all parts of the UI
@roblourens 

Sorry, but that is just a bad call from an accessibility perspective. Zooming also increases the size of buttons, panels, icons, padding, sometimes in ways that are pretty useless. (I’m generally not a fan of how VSCode handles that.)

And if you have quite bad eyes, there’s a point where you can’t zoom the whole window enough to see enough but still be productive, because there’s just a bunch of useless padding. The way people with bad eyes (including me) use VSCode is by zooming the app window a bit, but zooming the editor font A LOT. Usually, you just have to read few things in the UI (like the occasional file name if you don’t navigate via CMD+P). It’s fine to just get closer to the screen for a second there. But if you’re trying to read a long text or explanation by Copilot, it’s absolutely horrible. 

I don’t know what you mean by "The UI has quite a lot of text in it in some places". Nowhere in the UI do you read what’s essentially an essay about a potentially complex topic for potentially several minutes. It’s entirely different.

Please reconsider this (or actually, implementing a UI font size across VSCode would be the best thing). It’s a really bad call ergonomically and in terms of accessibility. I can’t work like this – I’m back to copy pasting my code into ChatGPT. 🤷‍♂️ 

Attached is a screenshot of what I’m talking about.

![image](https://github.com/microsoft/vscode/assets/34057943/6239203c-1bde-416c-86b1-0e4ef1f269ef)

**Comment by nbros** [comment]
I don't know why but currently the Chat view looks like it has a smaller font size than the rest of the UI. 
This is the only view I cannot read comfortably.

**Comment by doubleyewdee** [comment]
Just adding my $0.02 as an MSFT employee in VS Code every day (not all day, we have the meetings of course). I would really appreciate a solution for this accessibility concern.

Reiterating what @liam-k put so well:
> And if you have quite bad eyes, there’s a point where you can’t zoom the whole window enough to see enough but still be productive, because there’s just a bunch of useless padding. The way people with bad eyes (including me) use VSCode is by zooming the app window a bit, but zooming the editor font A LOT. Usually, you just have to read few things in the UI (like the occasional file name if you don’t navigate via CMD+P). It’s fine to just get closer to the screen for a second there. But if you’re trying to read a long text or explanation by Copilot, it’s absolutely horrible.

**Comment by qwordy** [comment]
Need it. I am almost blind viewing the chat window.

**Comment by gh0stintheshe11** [comment]
still didnt not add this setting after almost 2 years... Kudos, VSC team

**Comment by AMagicianNamedGob** [comment]
Instead of dealing with the "soution" of zoomin in and out, I'll just ctrl + tab over to Chat GPT's website. Maybe it's time to switch to Cursor permanantly.

**Comment by healingdrawing** [comment]
unsubscribed, it is useless noise

**Comment by lszomoru** [comment]
The latest VS Code Insiders release contains two new settings (`chat.fontFamily`, `chat.fontSize`) that allows you to control the font family/size. While we are still working on making sure that the various UI elements in the Chat panel respect the settings we would appreciate if you would give it a try and share your feedback. Thanks!

**Comment by alice-v252** [comment]
> The latest VS Code Insiders release contains two new settings (`chat.fontFamily`, `chat.fontSize`) that allows you to control the font family/size. While we are still working on making sure that the various UI elements in the Chat panel respect the settings we would appreciate if you would give it a try and share your feedback. Thanks!

It only changes code snippets in answers for me, not the "default" chat - the one used for "human-like" text

**Comment by lszomoru** [comment]
@alice-v252, what is the exact version of your VS Code? Could you also please share your chat settings? Thanks!

**Comment by BigBrotherJu** [comment]
@alice-v252 You have to use Insiders version.
